content/doc/developer/blueocean-plugin-development/index.adoc

```diff
@@ -0,0 +1,36 @@
+---
+layout: developerchapter
+summary: How to develop plugins for Blue Ocean
+wip: true
+title: Extending Blue Ocean
+references:
+- url: https://github.com/jenkinsci/blueocean-plugin
+  title: Main Blue Ocean plugin repository
+- url: https://github.com/jenkinsci/jenkins-design-language
+  title: Design language repository with the styles and common components for Blue Ocean
+---
+
+The Blue Ocean UX plugin for Jenkins is designed to be extended by other plugins. 
```

Comment:
s/Blue Ocean UX plugin/Blue Ocean plugin/


The plugin(s) don't self-identify as "Blue Ocean UX" so this might be confusing